pim-x-element.html

```diff
@@ -194,8 +205,20 @@
                         }
                     }
                 },
-
-                ready: function () {
+                
+                attached: function () {
+                    if (!Polymer.Element) {
+                        // Polymer version 1
+                        this.addData();
+                    } else {
+                        // Polymer version 2
+                        Polymer.RenderStatus.beforeNextRender(this, function() {
+                            this.addData();
+                        });
+                    }
+                },
```

Comment:
Does this only exist to support `console.log`? I might be wrong, but it looks like this can be simplified. I made a commit on a separate branch, please review: https://github.com/Juicy/x-breadcrumb/commit/909f318b351af4cd4ff5126376eacecbdcb789a5